zebra-chain/src/transaction/serialize.rs

```diff
@@ -672,6 +672,53 @@ impl ZcashSerialize for Transaction {
                 // `proofsOrchard`, `vSpendAuthSigsOrchard`, and `bindingSigOrchard`.
                 orchard_shielded_data.zcash_serialize(&mut writer)?;
             }
+
+            #[cfg(feature = "tx_v6")]
+            Transaction::V6 {
+                network_upgrade,
+                lock_time,
+                expiry_height,
+                inputs,
+                outputs,
+                sapling_shielded_data,
+                orchard_shielded_data,
+            } => {
+                // Transaction V5 spec:
+                // https://zips.z.cash/protocol/protocol.pdf#txnencoding
```

Comment:
```suggestion
                // Transaction V6 spec:
                // https://zips.z.cash/zip-0230#specification
```

I think this is the only reference we have to transaction v6 for now. All fields from tx v5 are present in v6, which is good for this code as it is a copy tx v5.